components/public-api-server/pkg/proxy/prometheusmetrics.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package proxy
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+func ReportConnectionDuration(d time.Duration) {
```

Comment:
```suggestion
func reportConnectionDuration(d time.Duration) {
```
doesn't need to be public